src/pages-and-resources/discussions/DiscussionAppCard.jsx

```diff
@@ -9,8 +9,8 @@ import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 
 import messages from './messages';
 
-function DiscussionToolOption({
-  intl, forum, selected, onSelect,
+function DiscussionAppCard({
```

Comment:
I forgot to rename this in my prior PR, which is why the filename already has this naming.